node/network/src/protocol.rs

```diff
@@ -73,6 +74,9 @@ const POISONED_LOCK_ERR: &str = "The lock was poisoned.";
 /// Time interval between printing connected peers.
 const CONNECTED_PEERS_INT: Duration = Duration::from_secs(30);
 
+/// Maximal number of blocks sent in one response.
+const MAX_NUM_BLOCKS_RESPONSE: u64 = 100;
```

Comment:
hm, would be nice probably to have this in config

Reply:
That's true. Maybe I should it in client config?

Reply:
`ClientConfig` is a perfect place. Not need to create an arg for it yet, just keep it in the config.